### PR TITLE
Drop couch support

### PR DESCRIPTION
**Pull request by sivakumar-kailasam**
- Remove couch code & dependencies
- Rewritten app adapter to work with static json-docs
- Updated FastBoot libs & ensured FastBoot compatibility
- Replaced ember-ajax with ember-network
- Removed legacy mirage folders & dependencies
- Added a yarn.lock file
- Task to sync the json-docs directory from s3
- Removed redis dependency

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -71,22 +68,20 @@
...
-    "ember-pouch": "github:gaurav0/ember-pouch#c89600c7b8c14c919b9b808a4ea6d9d6666ca047",
+    "ember-network": "0.3.1",
     "ember-resolver": "^2.0.3",
     "ember-route-action-helper": "2.0.0",
     "ember-source": "~2.11.2",
     "ember-truth-helpers": "1.2.0",
     "emberx-select": "github:thefrontside/emberx-select#05f0c947d321ca59e58cf1213a9552b888ca0201",
-    "fastboot-app-server": "1.0.0-rc.4",
+    "fastboot-app-server": "1.0.0-rc.5",
     "fastboot-redis-cache": "github:gaurav0/fastboot-redis-cache#16fa186392b66b1697608e582dd350483f1df722",
```
Comment (toddjordan) [inline package.json]: is the redis stuff needed now?

Comment (sivakumar-kailasam) [inline package.json]: Left it out so that we compare page load time with & without redis cache before removing it

**Comment by fivetanley** [comment]
:pray:

File: app/templates/components/class-field-description.hbs [hunk 2]
```diff
@@ -19,12 +19,15 @@
   <p class="github-link">
     {{#if field.inherited}}
       Inherited from
+      <a href={{github-link model.project.id model.projectVersion.version field.file field.line}} target="_blank">
```
Comment (toddjordan) [inline app/templates/components/class-field-description.hbs]: href value should be in quotes

Comment (sivakumar-kailasam) [inline app/templates/components/class-field-description.hbs]: Sure, should we add https://github.com/rwjblue/ember-template-lint?

File: app/templates/project-version/class.hbs [hunk 3]
```diff
@@ -26,7 +26,7 @@
       <div class="attribute">
         <span class="attribute-label">Defined in:</span>
         <span class="attribute-value">
-          <a href={{github-link model.project.id model.projectVersion.version model.file model.line}}>{{model.file}}:{{model.line}}</a>
+          <a href={{github-link model.project.id model.projectVersion.version model.file model.line}} target="_blank">{{model.file}}:{{model.line}}</a>
```
Comment (toddjordan) [inline app/templates/project-version/class.hbs]: href value in quotes

File: app/templates/properties.hbs [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {{ember-anchor a=anchor}}
 {{#api-index-filter model=model filterData=filterData as |filteredModel|}}
   {{#each filteredModel.properties as |property|}}
-    {{class-field-description type="property" field=property}}
+    {{class-field-description type="property" field=method model=model}}
```
Comment (toddjordan) [inline app/templates/properties.hbs]: field should be assigned `property` and not `method`

File: app/templates/events.hbs [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {{ember-anchor a=anchor}}
 {{#api-index-filter model=model filterData=filterData as |filteredModel|}}
   {{#each filteredModel.events as |event|}}
-    {{class-field-description type="event" field=event}}
+    {{class-field-description type="event" field=method model=model}}
```
Comment (toddjordan) [inline app/templates/events.hbs]: field should be assigned `event` and not method

**Comment by toddjordan** [comment]
addresses #133 and #138 

this looks good to go when you are ready.

**Pull request merged**